app/doc/responses.go

```diff
@@ -16,3 +16,16 @@ type updatedResponse struct {
 		Success bool `json:"success"`
 	}
 }
+
+// CreatedResponse returned when the request has successfully created the object
```

Comment:
The comment here is intended to be displayed in the doc, so it should the same format as the other. So "The request has successfully created the object". There is no lint check in the doc package anyway.

Reply:
Actually it is never displayed anywhere, so I removed it.